examples/reference-erc4626/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ponder-examples-reference-erc20",
```

Comment:
Will need to update this to `ponder-examples-reference-erc4626` to play nice with changesets.